packages/react-router-redux/modules/ConnectedRouter.js

```diff
@@ -26,7 +26,9 @@ class ConnectedRouter extends Component {
               payload: location
             })
 
-            return children
+            return !Array.isArray(children) || children.length ? ( // Preact defaults to empty children array
+              React.Children.only(children)
+            ) : null
```

Comment:
I'd rather use non-array semantics for this, as React maintains an opaque structure for children collections. Technically, it has a length property, but that may not always be that way in the future. 

Instead, we should take from [what `<Router>` does](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/modules/Router.js#L51-L54) and use `React.Children.count(children) === 1` (since it requires a single child anyways).